apps/frontend/src/app/modules/shared/components/card/card.component.html

```diff
@@ -1,14 +1,16 @@
-<div class="content" [ngStyle]="{ 'background-color': 'hsl(' + getColor(post.color) + ' 100% 80%)' }" [routerLink]="['/posts/' + post.id]">
-  <img class="index-img" [src]="post.index_image" alt="" />
-  <div class="label-container">
-    <verseghy-label *ngFor="let label of post.labels" [label]="label"></verseghy-label>
-  </div>
-  <p class="title">{{ post.title }}</p>
-  <p class="description" *ngIf="post.description">{{ post.description }}</p>
-  <div class="bottom">
-    <img *ngIf="post.author.image" class="author-img" [src]="post.author.image" alt="" />
-    <p>{{ post.author.name }}</p>
-    <p>{{ formatDate(post.date) }}</p>
-  </div>
-  <div class="focus-border"></div>
-</div>
+<a class="content" [ngStyle]="{ 'background-color': 'hsl(' + getColor(post.color) + ' 100% 80%)' }" [routerLink]="['/posts', post.id]">
+  <article>
```

Comment:
This is not an article, this is just a card that points the article, the article is in the posts page

Reply:
MDN suggests to use article for "product cards" too, so I think we should keep it. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article